README.md

```diff
@@ -32,6 +32,12 @@ brew install desktop-file-utils shared-mime-info       \
 
 
 ```sh
-meson setup build
-meson compile -C build
+git clone "https://github.com/gtkwave/gtkwave.git"
+cd gtkwave
+meson setup build && cd build && meson install
+```
+
+### Syntax to run vcd files:
+```
+gtkwave [your .vcd file path]
 ```
```

Comment:
This paragraph only mentions VCD files and only shows one way of running GTKWave from the command line. The other dump file formats and especially the option to run GTKWave with a `.gtkw` save file as a parameter are equally important. Perhaps it would be easier to link to the relevant docs and mark this as an example:
````suggestion
### Running GTKWave
```sh
gtkwave [path to a .vcd, .fst, .ghw dump file or a .gtkw savefile]
```
For more information about available command line parameters refer to the built in-help (`gtkwave --help`) or the (GTKWave docs)[https://gtkwave.github.io/gtkwave/man/gtkwave.1.html].
````